corehq/apps/case_search/utils.py

```diff
@@ -88,7 +99,8 @@ def _add_daterange_queries(self):
         pattern = re.compile(r'__range__\d{4}-\d{2}-\d{2}__\d{4}-\d{2}-\d{2}')
         drop_keys = []
         for key, val in self.criteria.items():
-            if val.startswith('__range__'):
+            # multiple daterange query param values are not supported
+            if not isinstance(val, list) and val.startswith('__range__'):
```

Comment:
should this raise an exception if it is a range param and also a list?:

Reply:
Yes, fixed that!